Makefile

```diff
@@ -166,56 +151,16 @@ cri-containerd:
 log-parser:
 	make -C cmd/log-parser
 
-oci:
-	systemctl is-active --quiet docker || sudo systemctl start docker
-	cd integration/oci_calls && \
-	bash -f oci_call_test.sh
-
 openshift:
 	bash -f .ci/install_bats.sh
 	bash -f integration/openshift/run_openshift_tests.sh
 
 pentest:
 	bash -f pentest/all.sh
 
-vm-factory:
-	bash -f integration/vm_factory/vm_templating_test.sh
-
-
-network:
-	systemctl is-active --quiet docker || sudo systemctl start docker
-	bash -f .ci/install_bats.sh
-	bats integration/network/macvlan/macvlan_driver.bats
-	bats integration/network/ipvlan/ipvlan_driver.bats
-	bats integration/network/disable_net/net_none.bats
-
-ramdisk:
-	bash -f integration/ramdisk/ramdisk.sh
-
-entropy:
-	bash -f .ci/install_bats.sh
-	bats integration/entropy/entropy_time.bats
-
-netmon:
-	systemctl is-active --quiet docker || sudo systemctl start docker
-	bash -f .ci/install_bats.sh
-	bats integration/netmon/netmon_test.bats
-
 tracing:
 	bash tracing/tracing-test.sh
 
-time-drift:
-	bats integration/time_drift/time_drift.bats
-
-compatibility:
-	bash -f integration/compatibility/run.sh
-
-vcpus:
```

Comment:
the vcpus integration test is being updated to use ctr so it should not be remove it

Reply:
Got it - added it back.